src/views/token-swap/components/amount-box/amount-box.view.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import useAmountBoxStyles from './amount-box.style'
+import { ReactComponent as AngleDown } from '../../../../images/icons/angle-down.svg'
+import FiatAmount from '../../../shared/fiat-amount/fiat-amount.view'
+import AmountInput from '../../../shared/amount-input/amount-input'
+import Dropdown from '../dropdown/dropdown.view'
+import { getTokenIcon } from '../../../../utils/tokens'
+import {
+  getTokenAmountInPreferredCurrency,
+  getFixedTokenAmount
+} from '../../../../utils/currencies'
+
+function AmountBox ({
+  account,
+  preferredCurrency,
+  fiatExchangeRates,
+  value,
+  amount,
+  onInputChange,
+  position,
+  accounts,
+  setToken,
```

Comment:
It's a good practice to use the `on...` prefix for event handlers. I would also avoid to use the `set` word and use the `change` for event handlers, as it's more suitable. For example: `onTokenChange` and `onActiveDropdownChange`.